// proto/messages.proto
syntax = "proto3";

service LoggingService {
  rpc LogMessage (LogRequest) returns (LogResponse);
  rpc GetMessages (Empty) returns (MessagesResponse);
}

message LogRequest {
  string id = 1;
  string message = 2;
}

message LogResponse {
  string status = 1;
}

message MessagesResponse {
  repeated string messages = 1;
}

message Empty {}
